homeworks/sasha.demidenko_SashaDemi/GamesProject/login.html

```diff
@@ -22,29 +22,31 @@
         </header>
         <main class="form">
             <h1 class="form_h1">Login page</h1>
-            <form action="success.html" class="form_content">
+            <form class="form_content">
                 <p class="form_p">
-                    <label for="email" class="form_email">Email</label>
+                    <label for="email-input" class="form_email">Email</label>
                     <br >
-                    <input type="email" id="email" name="email" required="required"
+                    <input id="email-input" name="email" required="required"
                         placeholder="Enter your email address" class="form_email--input">
                 </p>
                 <p class="form_p">
-                    <label for="pwd" class="form_pass">Password</label>
+                    <label for="password-input" class="form_pass">Password</label>
                     <br >
-                    <input type="password" id="pwd" name="pwd" required="required"
+                    <input id="password-input" name="pwd" required="required"
```

Comment:
Hey, @SashaDemi, the password field  should hide the entered text by default

Reply:
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/9cf43696-491d-43cc-8a6e-b69172aba388)

Reply:
please return `type="password"` to this input in the next task